test/IDE/coloring_52982457.swift

```diff
@@ -0,0 +1,115 @@
+// RUN: %target-swift-ide-test -syntax-coloring -source-filename %s | %FileCheck %s
```

Comment:
Could you add a comment at the top of the test to make it clear this is to check that the test case doesn't take a long time to finish?

Reply:
Done